dotCMS/src/integration-test/java/com/dotcms/content/elasticsearch/business/ESReadOnlyMonitorTest.java

```diff
@@ -37,7 +37,7 @@
 public class ESReadOnlyMonitorTest {
```

Comment:
This class should be renamed as the ESReadOnlyMonitor was removed and now the ElasticReadOnlyCommand is the class being tested